File: db4oj.tests/src/com/db4o/db4ounit/common/querying/AllTests.java
/* Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com */

package com.db4o.db4ounit.common.querying;

import db4ounit.extensions.*;

public class AllTests extends Db4oTestSuite {

	public static void main(String[] args) {
		new AllTests().runSoloAndClientServer();
    }

	protected Class[] testCases() {
		return new Class[] {
			IdListQueryResultTestCase.class,
            IndexedQueriesTestCase.class,
            MultiFieldIndexQueryTestCase.class,
            ObjectSetTestCase.class,
            OrderedQueryTestCase.class,
		};
	}
}

File: db4o.net/Db4objects.Db4o.Tests/Db4objects.Db4o.Tests/Common/Handlers/AllTests.cs
/* Copyright (C) 2004 - 2008  db4objects Inc.  http://www.db4o.com */

using System;
using Db4oUnit.Extensions;
using Db4objects.Db4o.Tests.Common.Handlers;

namespace Db4objects.Db4o.Tests.Common.Handlers
{
	public class AllTests : Db4oTestSuite
	{
		public static void Main(string[] args)
		{
			new Db4objects.Db4o.Tests.Common.Handlers.AllTests().RunAll();
		}

		protected override Type[] TestCases()
		{
			return new Type[] { typeof(ArrayHandlerTestCase), typeof(BooleanHandlerTestCase), 
				typeof(ByteHandlerTestCase), typeof(CharHandlerTestCase), typeof(ClassHandlerTestCase
				), typeof(CustomTypeHandlerTestCase), typeof(DoubleHandlerTestCase), typeof(FloatHandlerTestCase
				), typeof(IntHandlerTestCase), typeof(LongHandlerTestCase), typeof(MultiDimensionalArrayHandlerTestCase
				), typeof(SecondClassTestCase), typeof(StringBufferHandlerTestCase), typeof(StringHandlerTestCase
				), typeof(ShortHandlerTestCase), typeof(UntypedHandlerTestCase) };
		}
	}
}
